src/Generator/TraitUsageGenerator.php

```diff
@@ -54,9 +62,7 @@ public function addUse($use, $useAlias = null)
         return $this;
     }
 
-    /**
-     * @inheritDoc
-     */
+    /** {@inheritDoc} */
```

Comment:
Since this should be importing the whole docBlock, I would make it explicit https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc-tags.md#41-making-inheritance-explicit-using-the-inheritdoc-tag.

Reply:
dd68158de612598d53a7f3dcc96c27c5b3816dbf